test/LibraryManager.IntegrationTest/LibmanTestsUtility.cs

```diff
@@ -50,6 +70,13 @@ private static string WaitForCompletionEntriesHelper(IVisualStudioTextEditorTest
                         comparisonSet.Add(item.Text);
                     }
 
+                    // Make another call if it's still loading.
+                    if (comparisonSet.Count == 1 && comparisonSet.Contains("Loading..."))
```

Comment:
This is a localized string.  You can't hardcode product strings in tests.

Reply:
I see. I will check about it tomorrow.

---
In reply to: [193926293](https://github.com/aspnet/LibraryManager/pull/108#discussion_r193926293) [](ancestors = 193926293)

Reply:
Used the localized one defined in the vsix project

---
In reply to: [193935000](https://github.com/aspnet/LibraryManager/pull/108#discussion_r193935000) [](ancestors = 193935000,193926293)